packages/bits/demo/src/components/demo/pipes/unit-conversion/docs/unit-conversion-pipe-docs.component.html

```diff
@@ -0,0 +1,33 @@
+<p>
+    The <code><a href="../pipes/UnitConversionPipe.html#info" target="_parent">unitConversion</a></code> pipe can be used to convert a large value of a small
+    basic unit to a smaller value of a larger unit--for example, converting a quantity of 1024 in bytes to "1 KB". The converted unit label is automatically
+    appended to the output. Supported units are of type <code><a href="../miscellaneous/typealiases.html#UnitOption" target="_blank">UnitOption</a></code> and
+    include the following: <code>"generic"</code> (e.g. 1k for 1000), <code>"bytes"</code>, <code>"bytesPerSecond"</code>, <code>"bitsPerSecond"</code>, and
+    <code>"hertz"</code>.
+</p>
+
+<h2>Required Modules</h2>
+<ul>
+    <li>
+        <code>NuiCommonModule</code>
+    </li>
+</ul>
+
+<h2>Basic Usage</h2>
+<p>To get started with using the pipe, take a look at the template source of the example below.</p>
+
+<h3>Parameters</h3>
+<p>The pipe accepts the following parameters:</p>
+<ul>
+    <li><strong>value</strong> - The value to be converted.</li>
+    <li><strong>scale</strong> - The decimal scale of the formatted value--for specifying how many significant digits to display to the right of the decimal.</li>
```

Comment:
The same question here: shouldn't we have spaces between words? "value--for" -> "value - for"

Reply:
I'm using `--` as a crude approximation of an [em dash](https://www.thesaurus.com/e/grammar/em-dash/).

Reply:
But, in this case, I think you're right that it doesn't need it. So, I'll remove it. Thanks.